docs/source/tempest_pod.rst

```diff
@@ -0,0 +1,133 @@
+Run Tempest in a Pod
+====================
+Sometimes, you may want to avoid running the whole operator, e.g. for
+debugging or development purposes. In that case, you can run Tempest in a
+`pod <https://docs.openshift.com/online/pro/architecture/core_concepts/pods_and_services.html>`_.
+
+Write Object Definitions
+--------------------------
+
+Create a file named ``tempest-config.yaml`` with the following content:
+
+.. literalinclude:: samples/tempest-config.yaml
+   :language: yaml
+
+The file contains tempest configuration and will be used to pass any tempest
+options to the container running Tempest.
+
+Then create a file named ``tempest-deployment.yaml`` with the following content:
+
+.. literalinclude:: samples/tempest-deployment.yaml
+   :language: yaml
+
+The file contains a pod definition - it tells OpenShift to create a Pod running
+one container that will run Tempest from the openstack-tempest image. More about
+images can be found `here <./images.html>`_.
+
+Create Resources
+----------------
+
+Run the `oc apply` command to create the resources:
+
+.. code-block:: bash
+
+    oc apply -f tempest-config.yaml
+    oc apply -f tempest-deployment.yaml
```

Comment:
Just an observation: Isn't weird that the tempest-worker pod finishes and then it is started again because openshift detects the end of the script as a Crash?

Reply:
really? does it detect the end of the script as a fail? .. that is a bug then, let's track it separately

Reply:
Can't reproduce now. Previously the pod was oscillating between CrashLoopBackOff <-> Running. Now when I try to reproduce it, the pod status keeps changing between Completed <-> Running. I will keep an eye on this.